### ISSUE TITLE
`field_class` option on form level is ignored in select lists

### ISSUE THREAD

**Issue by protaskz-in** [shivam091/rails_bootstrap_form#44]
Hi.

I come across one issue and found that `field_class` is ignored in select list when specified at form level:

```erb
<%= bootstrap_form_for @user, bootstrap: {field_class: "form-control"} do |form| %>
  <%= form.fields_for :address, include_id: false do |address_form| %>
    <%= address_form.select :country_id, options_for_select(::Country.pluck(:name, :id), address_form.object.country_id),
        {include_blank: "Select Country"} %>
  <% end %>
<% end %>
```

and corresponding HTML:

```html
<form role="form" novalidate="novalidate" class="new_user" id="new_user" action="/users" accept-charset="UTF-8" method="post">
  <div class="mb-3">
    <label class="form-label required" for="user_address_attributes_country_id">Country</label>
    <select class="form-select" aria-required="true" required="required" name="user[address_attributes][country_id]" id="user_address_attributes_country_id">
      <option value="">Select Country</option>
      <option value="1">India</option>
      <option value="2">Ireland</option>
      <option value="3">United States</option>
      <option value="4">United Kingdom</option>
      <option value="5">Spain</option>
      <option value="6">France</option>
      <option value="7">Canada</option>
    </select>
  </div>
</form>
```

**Comment by shivam091** [comment]
Hi.

Thank you for reporting this.

**Comment by shivam091** [comment]
Hello,

This will get fixed in upcoming release `0.9.5`.
